__tests__/app.test.js

```diff
@@ -0,0 +1,36 @@
+const db = require("../db/connection");
+const app = require("../app");
+const data = require("../db/data/test-data");
+const seed = require("../db/seeds/seed");
+const request = require("supertest");
+
+beforeEach(() => seed(data));
+
+afterAll(() => {
+  return db.end();
+});
+
+describe("Testing endpoints in the app.js file", () => {
+  test("GET: 200 - request to api/topcs returns all available topics", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((response) => {
+        const topics = response.body.topics;
+        topics.forEach((topic) => {
+          expect(topic).toEqual(
+            expect.objectContaining({
+              description: expect.any(String),
+              slug: expect.any(String),
+            })
+          );
+        });
+      });
```

Comment:
This is good except that if the returned array is empty then your `forEach` loop would be skipped, and your test assertions skipped along with it resulting in a false positive ( jest tests with no assertions default to pass ) 

Consider adding a check for the expected length of the `topics` before entering a `forEach` loop